addOns/network/src/main/java/org/zaproxy/addon/network/GlobalExclusionsOptions.java

```diff
@@ -188,7 +212,49 @@ protected void updateConfigsImpl(int fileVersion) {
                     setGlobalExclusions(DEFAULT_GLOBAL_EXCLUSIONS);
                     break;
                 }
+            case 1:
+                updateToVersion2();
+        }
+    }
+
+    private void updateToVersion2() {
+        List<HierarchicalConfiguration> fields =
+                ((HierarchicalConfiguration) getConfig()).configurationsAt(EXCLUSION_KEY);
+        globalExclusions = new ArrayList<>(fields.size());
+        List<String> oldValues = new ArrayList<>(BEFORE_V2_UPDATE.size());
+        BEFORE_V2_UPDATE.forEach(e -> oldValues.add(e.getValue()));
+        for (HierarchicalConfiguration sub : fields) {
+            try {
+                String value = sub.getString(GLOBAL_EXCLUSION_VALUE_KEY, "");
+                String name = sub.getString(GLOBAL_EXCLUSION_NAME_KEY, "");
+                if (oldValues.contains(value)) {
+                    // Fix naming mistake
+                    if (name.contains("adx")) {
+                        name = name.replace("adx", "axd");
+                    }
+                    // Extend image types pattern(s)
+                    if (value.contains("|bmp)")) {
+                        value = value.replace("|bmp)", "|bmp|svg|webp)");
+                    }
+                    // Extend audio/video types pattern(s)
+                    if (value.contains("|og[gav])")) {
+                        value = value.replace("|og[gav])", "|og[gav]|webm)");
+                    }
+                    // Make them case insensitive
+                    value = "(?i)" + value;
```

Comment:
Should set the value (and the name where needed) into the sub config rather than validating/reparsing/setting the entries (that's already going to be done later).